app/models/comment.rb

```diff
@@ -1,4 +1,6 @@
 class Comment < ApplicationRecord
   belongs_to :request
   belongs_to :author, polymorphic: true
+
+  validates :content, presence: true
```

Comment:
you can add author and request, no? -> `validates :content, :author, :request, presence: true`